src/reducers/authReducer.js

```diff
@@ -0,0 +1,25 @@
+export const defaultAuthState = {
+  user: {},
+  loggedIn: 'false',
```

Comment:
Is this intended to be a _string_ type?

Reply:
Not at all, thanks for catching that. Helped clean a lot of the issues I was having.